iset.mm

```diff
@@ -30751,11 +30768,11 @@ intersection or the difference (that is, this theorem would be equality
   $}
 
   ${
-    ifcldadc.1 $e |- ( ( ph /\ ps ) -> A e. C ) $.
-    ifcldadc.2 $e |- ( ( ph /\ -. ps ) -> B e. C ) $.
-    ifcldadc.dc $e |- ( ph -> DECID ps ) $.
+    ifcldacd.1 $e |- ( ( ph /\ ps ) -> A e. C ) $.
```

Comment:
What are you thinking with this naming? `ifcldadc` is `ifclda` from set.mm plus `dc`. I've often wondering whether it should be `ifcldcda` but I don't know where `ifcldacd` is coming from.

Reply:
I'm not sure what happened in my mind... I'm going to revert !

Reply:
I could try to guess what happened (`ifcldadc` is a bit of a strange name), but reverting sounds fine.

Reply:
Done.